deploy-manage/security/self-tls-considerations.md

```diff
@@ -0,0 +1,37 @@
+---
+navigation_title: Considerations
+applies_to:
+  deployment:
+    self:
+    eck:
+products:
+  - id: elasticsearch
+navigation_title: Private or 3P CA considerations
+---
+
+# Considerations for using an private or third-party CA for transport layer security
+
+By default, {{es}} uses mutual TLS (mTLS) to secure node-to-node transport connections. Mutual TLS means that data is encrypted in transit, ensuring confidentiality and integrity, and also that both nodes in a connection must present a valid certificate to the other node when establishing the connection. Each node requires that certificates be issued by a trusted certificate authority, ensuring that only authorized nodes can connect. Configure trusted certificate authorities using settings in the [`xpack.security.transport.ssl.*`](elasticsearch://reference/elasticsearch/configuration-reference/security-settings.md#transport-tls-ssl-settings) namespace, such as `xpack.security.transport.ssl.certificate_authorities` and `xpack.security.transport.ssl.truststore.path`. 
+
```

Comment:
Maybe we should add here some more context about the different kinds of CA: Elasticsearch comes with the `elasticsearch-certutil` tool that allows you to create a properly-configured private CA and issue certificates for the nodes in the cluster, but it is also compatible with certificates obtained from a different CA managed using other tools. This page is about the latter case.